helm_deploy/hmpps-education-and-work-plan-api/values.yaml

```diff
@@ -37,6 +37,14 @@ generic-service:
       DB_USER: "database_username"
       DB_PASS: "database_password"
 
+    # Inbound SQS config
+    hmpps-domain-events-topic:
+      HMPPS_SQS_TOPICS_DOMAINEVENTS_ARN: "topic_arn"
+    education-and-work-plan-domain-events-sqs-instance-output:
+      HMPPS_SQS_QUEUES_EDUCATION_AND_WORK_PLAN_QUEUE_NAME: "sqs_queue_name"
+    education-and-work-plan-domain-events-sqs-dl-instance-output:
+      HMPPS_SQS_QUEUES_EDUCATION_AND_WORK_PLAN_DLQ_NAME: "sqs_queue_name"
```

Comment:
This is all magic! The names have to be exact and I won't know if this is definitely correct until we deploy it to dev. However, I've looked at other repos and I'm hoping this is right 🤞

Reply:
Do you need to do anything to our namespace in CPE to provide these secrets? An `irsa.tf` file like they have in `hmpps-activities-management-dev` ?
(I have no idea what IRSA stands for? 🤷‍♂️ )

Reply:
Ah yes, good point. I'll create a PR for that 👍

Reply:
https://github.com/ministryofjustice/cloud-platform-environments/pull/16703